charts/pocketd-faucet/templates/deployment.yaml

```diff
@@ -0,0 +1,103 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "pocketd-faucet.fullname" . }}
+  labels:
+    {{- include "pocketd-faucet.labels" . | nindent 4 }}
+spec:
+  {{- if not .Values.autoscaling.enabled }}
+  replicas: {{ .Values.replicaCount }}
+  {{- end }}
+  selector:
+    matchLabels:
+      {{- include "pocketd-faucet.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "pocketd-faucet.selectorLabels" . | nindent 8 }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      #serviceAccountName: {{ include "pocketd-faucet.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      #initContainers:
+      #  - name: init-keyring
+      #    image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+      #    imagePullPolicy: {{ .Values.image.pullPolicy }}
+      #    command:
+      #      - sh
+      #      - -c
+      #      - |
+      #        echo -n "{{ .Values.env.SIGNING_KEY_MNEMONIC }}" > /mnemonic.txt
+      #        pocketd keys add \
+      #        {{ .Values.env.FAUCET_SIGNING_KEY_NAME }} \
+      #        --recover --source=/mnemonic.txt \
+      #        --keyring-backend={{ .Values.env.KEYRING_BACKEND }}
+      #      #--home={{/* .Values.homeDirectory */}}
+      containers:
+        - name: {{ .Chart.Name }}
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          command:
+            - sh
+            - -c
+            - |
+              echo -n "{{ .Values.env.SIGNING_KEY_MNEMONIC }}" > /mnemonic.txt
+              pocketd keys add \
+              {{ .Values.env.FAUCET_SIGNING_KEY_NAME }} \
+              --recover --source=/mnemonic.txt \
```

Comment:
While this will work, it is much safer to mount the key file from the secret like we do this in pocketd binary: https://github.com/pokt-network/helm-charts/blob/main/charts/pocketd/templates/StatefulSet.yaml#L385

If we need to get it out ASAP - we can switch to a proper secret later.

Reply:
I've added TODOs to track this. I wasn't prepared to deal with it quite yet. :sweat_smile: